### PR TITLE
German: Use Konjuration instead of Hervorrufung for conjuration

### PR DESCRIPTION
**Pull request by powerbf**

### CODE REVIEW

File: crawl-ref/source/dat/descript/de/gods.txt [hunk 1]
```diff
@@ -667,7 +667,7 @@ Vehumet wrath
 
 Wer Vehumet erzürnt, wird mit großer Vernichtung rechnen müssen. Novizen werden
 von Flammen und Frost getroffen, Erzmagier von Kristallspeeren und Feuerstürmen.
-Auch ihre eigenen Hervorrufungen gehen oft im ungünstigsten Moment fehl.
+Auch ihre eigenen Konjurationen gehen oft im ungünstigsten Moment fehl.
```
Comment (Talandazar) [inline crawl-ref/source/dat/descript/de/gods.txt:670]: Auch ihre eigenen Konjurationen schlagen oft im ungünstigsten Moment fehl.

File: crawl-ref/source/dat/descript/de/hints.txt [hunk 2]
```diff
@@ -919,7 +919,7 @@ du auch dessen Beschreibung lesen.</tiles>
 %%%%
 HINT_ATTACK_MONSTER_MAGIC
 
-Als Hervorrufer wirst du es jedoch mit Magie behandeln wollen. Im
+Als Konjurant wirst du es jedoch mit Magie behandeln wollen. Im
```
Comment (Talandazar) [inline crawl-ref/source/dat/descript/de/hints.txt:922]: Not sure about this sentence. I don't see HINT_ATTACK_MONSTER_MAGIC in:
/crawl-ref/source/dat/descript/hints.txt

Probably this is better:
Als Konjurant solltest du jedoch Magie bevorzugen.

File: crawl-ref/source/dat/descript/de/hints.txt [hunk 3]
```diff
@@ -919,7 +919,7 @@ du auch dessen Beschreibung lesen.</tiles>
...
 Hilfe-Eintrag für den Memorierungsreiter (<w>$cmd[CMD_MEMORISE_SPELL]</w>)
```
Comment (Talandazar) [inline crawl-ref/source/dat/descript/de/hints.txt:923]: Memorierungsreiter does not work. Memorierung isn't a thing. It should be:

Auswendig lernen
and
"Auswendig lernen"-Reiter

File: crawl-ref/source/dat/strings/de/monsters-plural-counted-nom.txt [hunk 4]
```diff
@@ -1660,7 +1660,7 @@
 %d Oklob-Setzling-Zauberer
 %%%%
 %d oklob plant conjurers
-%d Oklob-Pflanze-Hervorrufer
+%d Oklob-Pflanze-Konjurant
```
Comment (Talandazar) [inline crawl-ref/source/dat/strings/de/monsters-plural-counted-nom.txt:1663]: Oklobpflanzenkonjurant

File: crawl-ref/source/dat/strings/de/monsters-plural-counted-nom.txt [hunk 5]
```diff
@@ -1660,7 +1660,7 @@
 %d Oklob-Setzling-Zauberer
```
Comment (Talandazar) [inline crawl-ref/source/dat/strings/de/monsters-plural-counted-nom.txt:1660]: Oklobsetzlingszauberer

File: crawl-ref/source/dat/strings/de/monsters-plural-counted-nom.txt [hunk 6]
```diff
@@ -1660,7 +1660,7 @@
...
 %%%%
 %d oklob plant conjurers
-%d Oklob-Pflanze-Hervorrufer
+%d Oklob-Pflanze-Konjurant
 %%%%
 %d oklob plant summoners
 %d Oklob-Pflanze-Beschwörer
```
Comment (Talandazar) [inline crawl-ref/source/dat/strings/de/monsters-plural-counted-nom.txt:1666]: Oklobpflanzenbeschwörer

File: crawl-ref/source/dat/strings/de/monsters-singular-def-nom.txt [hunk 7]
```diff
@@ -1676,7 +1676,7 @@ the %soklob sapling wizard
 der {adj-e}%sOklob-Setzling-Zauberer
 %%%%
 the %soklob plant conjurer
-der {adj-e}%sOklob-Pflanze-Hervorrufer
+der {adj-e}%sOklob-Pflanze-Konjurant
```
Comment (Talandazar) [inline crawl-ref/source/dat/strings/de/monsters-singular-def-nom.txt:1679]: dito

File: crawl-ref/source/dat/strings/de/monsters-singular-def-poss.txt [hunk 8]
```diff
@@ -1674,7 +1674,7 @@ the %soklob sapling wizard's
 des {adj-en}%sOklob-Setzling-Zauberers
 %%%%
 the %soklob plant conjurer's
-des {adj-en}%sOklob-Pflanze-Hervorrufers
+des {adj-en}%sOklob-Pflanze-Konjuranten
```
Comment (Talandazar) [inline crawl-ref/source/dat/strings/de/monsters-singular-def-poss.txt:1677]: dito

File: crawl-ref/source/dat/strings/de/monsters-singular-indef-nom.txt [hunk 9]
```diff
@@ -1674,7 +1674,7 @@ a %soklob sapling wizard
 ein {adj-er}%sOklob-Setzling-Zauberer
 %%%%
 a %soklob plant conjurer
-ein {adj-er}%sOklob-Pflanze-Hervorrufer
+ein {adj-er}%sOklob-Pflanze-Konjurant
```
Comment (Talandazar) [inline crawl-ref/source/dat/strings/de/monsters-singular-indef-nom.txt:1677]: dito

File: crawl-ref/source/dat/strings/de/monsters-singular-indef-poss.txt [hunk 10]
```diff
@@ -1675,7 +1675,7 @@ a %soklob sapling wizard's
 eines {adj-en}%sOklob-Setzling-Zauberers
 %%%%
 a %soklob plant conjurer's
-eines {adj-en}%sOklob-Pflanze-Hervorrufers
+eines {adj-en}%sOklob-Pflanze-Konjuranten
```
Comment (Talandazar) [inline crawl-ref/source/dat/strings/de/monsters-singular-indef-poss.txt:1678]: dito

File: crawl-ref/source/dat/strings/de/monsters-singular-plain-nom.txt [hunk 11]
```diff
@@ -1674,7 +1674,7 @@
 {adj-er}%sOklob-Setzling-Zauberer
 %%%%
 %soklob plant conjurer
-{adj-er}%sOklob-Pflanze-Hervorrufer
+{adj-er}%sOklob-Pflanze-Konjurant
```
Comment (Talandazar) [inline crawl-ref/source/dat/strings/de/monsters-singular-plain-nom.txt:1677]: dito

File: crawl-ref/source/dat/strings/de/monsters-singular-your-nom.txt [hunk 12]
```diff
@@ -1674,7 +1674,7 @@ your %soklob sapling wizard
 dein {adj-er}%sOklob-Setzling-Zauberer
 %%%%
 your %soklob plant conjurer
-dein {adj-er}%sOklob-Pflanze-Hervorrufer
+dein {adj-er}%sOklob-Pflanze-Konjurant
```
Comment (Talandazar) [inline crawl-ref/source/dat/strings/de/monsters-singular-your-nom.txt:1677]: dito

File: crawl-ref/source/dat/strings/de/monsters-singular-your-poss.txt [hunk 13]
```diff
@@ -1675,7 +1675,7 @@ your %soklob sapling wizard's
 deines {adj-en}%sOklob-Setzling-Zauberers
 %%%%
 your %soklob plant conjurer's
-deines {adj-en}%sOklob-Pflanze-Hervorrufers
+deines {adj-en}%sOklob-Pflanze-Konjuranten
```
Comment (Talandazar) [inline crawl-ref/source/dat/strings/de/monsters-singular-your-poss.txt:1678]: dito

**Review by Talandazar — APPROVED** [review]
Works great overall. Commented on changes I'd make.

File: crawl-ref/source/dat/descript/de/hints.txt (hunk 2 above)
Comment (powerbf) [reply crawl-ref/source/dat/descript/de/hints.txt:922]: I checked. It's there in the English file.

%%%%
HINT_ATTACK_MONSTER_MAGIC

However, as a conjurer you will want to deal with it using magic. If you look at
the help entry for the <w>M</w>emorisation screen you'll find an explanation of
how to do this.
%%%%

File: crawl-ref/source/dat/strings/de/monsters-plural-counted-nom.txt (hunk 4 above)
Comment (powerbf) [reply crawl-ref/source/dat/strings/de/monsters-plural-counted-nom.txt:1663]: There's a lot of this. I'll handle it in a separate PR.

**Pull request merged**